### ISSUE TITLE
Drag gesture doesn't follow the movement

### ISSUE THREAD

**Issue by appcornerit** [antoniocasero/Panels#9]
Hi, 
great project to reproduce Android bottom sheets.
It would be great to have a drag gesture that follow the movement, not just a toggle up/down.
Best.

**Comment by antoniocasero** [comment]
Yep, this is something that I have in mind but I need to find time to implement it 😄

**Comment by fanckush** [comment]
hello, any chance to implement this enhancement anytime soon :D
